src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -93,6 +100,11 @@ export class GcsApiService extends GcsService {
     }
   }
 
+  public cancelExportToS3() {
+    this.electronService.ipcRenderer.send('download-cancel');
```

Comment:
This doesn't make sense to me; don't we want to do this only after the dialog is shown?